conans/model/build_info.py

```diff
@@ -79,7 +79,7 @@ def __getitem__(self, item):
         return self._dependencies[item]
 
     def __repr__(self):
-        return TXTGenerator(self).content
+        return TXTGenerator(self, CppInfo(os.getcwd())).content
```

Comment:
The fix is OK, but I'd say this is a small code smell, i.e. a previous code smell, not in this PR. I am going to check the usage of this _ _ repr_ _()